### ISSUE TITLE
Cannot install any version past 0.3.0

### ISSUE THREAD

**Issue by FedericoStra** [JuliaMath/AccurateArithmetic.jl#25]
Probably related to #23:

```julia
julia> versioninfo()
Julia Version 1.6.0
Commit f9720dc2eb (2021-03-24 12:55 UTC)
Platform Info:
  OS: Linux (x86_64-redhat-linux)
  CPU: Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-11.0.1 (ORCJIT, skylake)

(@v1.6) pkg> add AccurateArithmetic @0.3.5
   Resolving package versions...
ERROR: Unsatisfiable requirements detected for package SIMDPirates [21efa798]:
 SIMDPirates [21efa798] log:
 ├─possible versions are: 0.1.0-0.8.26 or uninstalled
 ├─restricted by julia compatibility requirements to versions: uninstalled
 └─restricted by compatibility requirements with AccurateArithmetic [22286c92] to versions: [0.2.0-0.3.16, 0.5.0, 0.6.3-0.8.26] — no versions left
   └─AccurateArithmetic [22286c92] log:
     ├─possible versions are: 0.2.1-0.3.5 or uninstalled
     └─restricted to versions 0.3.5 by an explicit requirement, leaving only versions 0.3.5

(@v1.6) pkg> add AccurateArithmetic @0.3.4
    Updating registry at `~/.julia/registries/General`
   Resolving package versions...
ERROR: Unsatisfiable requirements detected for package SIMDPirates [21efa798]:
 SIMDPirates [21efa798] log:
 ├─possible versions are: 0.1.0-0.8.26 or uninstalled
 ├─restricted by julia compatibility requirements to versions: uninstalled
 └─restricted by compatibility requirements with AccurateArithmetic [22286c92] to versions: [0.2.0-0.3.16, 0.5.0, 0.6.3-0.8.26] — no versions left
   └─AccurateArithmetic [22286c92] log:
     ├─possible versions are: 0.2.1-0.3.5 or uninstalled
     └─restricted to versions 0.3.4 by an explicit requirement, leaving only versions 0.3.4

...

(@v1.6) pkg> add AccurateArithmetic
   Resolving package versions...
    Updating `~/.julia/environments/v1.6/Project.toml`
  [22286c92] + AccurateArithmetic v0.3.0
    Updating `~/.julia/environments/v1.6/Manifest.toml`
  [22286c92] + AccurateArithmetic v0.3.0
```

**Comment by ffevotte** [comment]
Yes, sorry. It is indeed related to #23, and @chriselrod made almost all the hard work needed to make everything compatible with Julia 1.6 in #24, which is extended in #26.

We'll try and release this ASAP but, in the meantime, you should still be able to use most of `AccurateArithmetic.jl` with Julia 1.6, using:
```
pkg> add AccurateArithmetic#newvecbase
```

**Comment by ffevotte** [comment]
Version 0.3.6 should now be released in the General registry and available for installation with Julia v1.6
